website/docs/basics/writing-styles.md

```diff
@@ -7,7 +7,7 @@ title: Writing Styles
 
 > This chapter is only relevant in setups that don't use CSS variables. If you are using CSS variables please skip to the [Blocks Styles chapter](blocks-styles) chapter.
```

Comment:
Maybe we should move the legacy sections away from main docs, not to confuse people?